django_comments/forms.py

```diff
@@ -137,8 +137,12 @@ def get_comment_create_data(self, site_id=None):
         custom comment apps that override get_comment_model can override this
         method to add extra fields onto a custom comment model.
         """
+        COMMENTS_FOR_CONCRETE_MODEL = getattr(settings, 'COMMENTS_FOR_CONCRETE_MODEL', True)
         return dict(
-            content_type=ContentType.objects.get_for_model(self.target_object),
+            content_type=ContentType.objects.get_for_model(
+                self.target_object,
+                for_concrete_model=COMMENTS_FOR_CONCRETE_MODEL,
+            ),
```

Comment:
I'm not sure if the new setting is the preferred way :thinking: What do you think about adding the `for_concrete_model=True` keyword argument to the `get_comment_object()` and `get_comment_create_data()`?